components/home/WorkoutContainer.jsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { View, StyleSheet } from 'react-native';
+import { Button, Subheading } from 'react-native-paper';
+
+const buttonStartLocation = 28;
+const textStartLocation = 30;
+
+export default (props) => (
+  <View>
+    {props.items.map((item, index) => (
+      <Button contentStyle={{ width: 100, height: 35 }} style={{ ...styles.startButton, top: buttonStartLocation + index * 70 }} color="#FFFFFF" onPress={() => alert(`This should start recording ${item.name}`)}>
+        Start
+      </Button>
+    ))}
+
+    {props.items.map((item, index) => (
+      <Subheading style={[{
+        position: 'absolute', top: textStartLocation + index * 70, left: 15, fontSize: 18,
```

Comment:
Also is there a reason you chose to absolutely position these elements, I think it makes more sense to have them be relatively positioned but if this works that's just a personal preference kinda thing

Reply:
Personal preference for it I guess for me too. If it becomes an issue when I implement the functionality, I'll change it to relative positioning